### PR TITLE
Fix #1135 issues

### LINKED ISSUE #1135: Docs.tizen.org issues
**Please check the hyperlink.** 

1. https://docs.tizen.org/platform/what-is-tizen/overview/ : Tizen Store 
2. https://docs.tizen.org/platform/what-is-tizen/versions/tizen-5-5-m2/ : IoT Partners
3. https://docs.tizen.org/platform/porting/system/ : dump_systemstate
4. https://docs.tizen.org/platform/porting/multimedia/ 
: http://v4l2spec.bytesex.org/spec-single/v4l2.html
5. https://docs.tizen.org/platform/porting/connectivity/
: http://linuxwireless.org/
6. https://docs.tizen.org/platform/release-notes/tizen-5-5-m1/
: http://tizen.org/feature/accessibility.grayscale
: http://tizen.org/feature/accessibility.negative
7. https://docs.tizen.org/application/dotnet/guides/nui/animation/ 
: Download the _sample file_.
**Need to report to fix this**
8. https://docs.tizen.org/application/dotnet/guides/nui/xaml/layout-define-example/
: _XAML Support for Tizen.NUI_. 
9. https://docs.tizen.org/application/web/guides/connectivity/iotcon
: IoTivity
: see _Iotivity Security_.
: see _Iotivity Security_.
10. https://docs.tizen.org/application/native/guides/ui/dali/animation/#overview
: occur in a dedicated thread
11. https://docs.tizen.org/application/native/guides/multimedia/image-edit/
: see _http://www.fourcc.org/yuv.php_
: _RGB, RGBA_ of Table: RGB pixel formats
: _UYVY, YUYV_ of Table: Packed YUV formats
: _YV16, YV12, I420, NV12, NV21_ of Table: Planar YUV formats
12. https://docs.tizen.org/application/native/guides/connectivity/iotcon/
: _IoTivity_ offers 
13. https://docs.tizen.org/application/native/guides/personal/context/
: see the _Sound Manager_
14. https://docs.tizen.org/application/tizen-studio/native-tools/manifest-text-editor/
: Manifest Element Hierarchy  (overall)
15. https://docs.tizen.org/application/tizen-studio/common-tools/emulator-features
: see _QEMU Emulator User Documentation_
16. https://docs.tizen.org/application/tizen-studio/web-tools/ide-preferences
: is _http://yourdomain/_
17. https://docs.tizen.org/application/tizen-studio/web-tools/config-editor/
: Configuration Element Hierarchy (overall)
: through the Tizen _Application_ API. (to latest)
: through the Tizen Application API. (rewrite)
: <tizen:category name=_“http://tizen.org/category/ime”_/> (delete hyperlink)
18. https://docs.tizen.org/application/tizen-studio/web-tools/web-localization
: see _Localization_
19. https://docs.tizen.org/application/vstools/tools/manifest-editor/
: Manifest Element Hierarchy (overall)

**Please check all outdated hyperlinks to https://www.w3.org/.**
1. https://docs.tizen.org/application/web/guides/w3c/w3c-overview/
: All documents of W3C/HTML5/Supplementary Features
2. https://docs.tizen.org/application/tizen-studio/web-tools/config-editor/
: see _Widget Packaging and XML Configuration_
: in the _Widget Packaging and XML Configuration guidelines_
: in the _Widget Packaging and XML Configuration guidelines_

**Please check the unnecessary sentence.**
1. https://docs.tizen.org/platform/what-is-tizen/versions/tizen-4-0-m2/
: For more information of Tizen IoT preview 1, see here.

**Please check the TOC box.**
1. https://docs.tizen.org/platform/get-started/git-repo-structure/

**Please check the style consistency.**
Font style
1. https://docs.tizen.org/platform/what-is-tizen/profiles/tv/
: Create Samsung Smart TV Web applications
2. https://docs.tizen.org/platform/what-is-tizen/versions/tizen-4-0-m2/
: Figure. Evolution of Tizen
: Figure. Tizen .NET architecture and groundwork
3. https://docs.tizen.org/application/native/api/overview/
: Figure:Required Header
: Figure:Related Feature
: Figure:Reference Structure
: Figure:Privilege

Note style
1. https://docs.tizen.org/application/dotnet/guides/nui/styling-controls-with-JSON/
: Note in the Table: View style properties affecting a push button

**Please insert the proper image.**
1. https://docs.tizen.org/application/tizen-studio/setup/install-sdk
: Click the _…_ button 
cf. advanced_conf_browse.png of https://docs.tizen.org/application/tizen-studio/setup/advanced-configuration

**Please check broken images.**
1. https://docs.tizen.org/application/web/get-started/wearable/first-app/
: Select the profile (Wearable) and version from a drop-down list and click Next.
: Select the Web Application application type and click Next.
2. https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Box.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Box.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Conformant.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Conformant.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Layout.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Layout.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Mapbuf.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Mapbuf.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Panes.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Bg.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Ctxpopup.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Label.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__List.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Radio.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Slider.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Win.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Bg.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Colorselector.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Ctxpopup.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Hoversel.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Label.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__List.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Map.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Notify.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Panel.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Photocam.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Radio.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Segment__Control.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Slider.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Spinner.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Toolbar.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Win.html
: _preview-00.png_

3. https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Button.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Check.html
https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Datetime.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Button.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Check.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Datetime.html
: _preview-00.png, preview-01.png, preview-02.png_

4. https://docs.tizen.org/application/native/api/wearable/latest/group__Elm__Entry.html
https://docs.tizen.org/application/native/api/mobile/latest/group__Elm__Entry.html
: _preview-00.png, preview-01.png, preview-02.png, preview-03.png_

**Please check TOC missing titles.**
1. https://docs.tizen.org/application/web/guides/connectivity/connectivity/
: Network Bearer Selection
2. https://docs.tizen.org/application/web/guides/sensors/sensors/
: Gesture Recognition
3. https://docs.tizen.org/application/web/guides/text-input/text-input/
: Voice Control
4. https://docs.tizen.org/application/dotnet/guides/connectivity/overview/
: Internet and Content Downloads
5. https://docs.tizen.org/application/native/tutorials/overview/
: Designing Your Native Application

**Please check Overview missing titles.**
1. https://docs.tizen.org/application/dotnet/guides/overview/
: OpenTK
2. https://docs.tizen.org/application/dotnet/guides/connectivity/overview/
: Connection Management
: Web view
3. https://docs.tizen.org/application/web/guides/w3c/perf-opt/performance-guide/
: JavaScript Performance Improvement
: JavaScript and CSS Minification
: jQuery Performance Improvement

**Please check the TOC tree.**
1. Media Content and Metadata, Media Content, Metadata
https://docs.tizen.org/application/dotnet/guides/multimedia/media-content-metadata/
https://docs.tizen.org/application/dotnet/guides/multimedia/media-content/
https://docs.tizen.org/application/dotnet/guides/multimedia/metadata/

### PR DESCRIPTION
**Pull request by mijin85cho**
Signed-off-by: Mijin, Cho <mijin85.cho@samsung.com>

### Change Description ###

Among the issues reported by @mijin80lee,
I fixed the issues that caused by tizen-docs/docs files.

**Request :** 
There are some links that goes to pages external site.
Some broken links are caused by these as the external site URLs are changed.

I tried catch up the updated URLs, but I'm not sure with some pages.

If you can get the correct updated link, please let me know.

<table>
    <thead>
        <tr>
            <td rowspan="2"> Issued page</td>
            <td><em>Issued URL</em></td>
        </tr>
        <tr>
            <td> => Updated URL</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td rowspan="2"> /platform/porting/multimedia.md</td>
            <td><em>http://v4l2spec.bytesex.org/spec-single/v4l2.html</em></td>
        </tr>
        <tr>
            <td> => https://linuxtv.org/downloads/legacy/video4linux/API/V4L2_API/</td>
        </tr>
                <tr>
            <td rowspan="2"> /platform/porting/connectivity.md</td>
            <td><em>http://linuxwireless.org</em></td>
        </tr>
        <tr>
            <td> => https://wireless.wiki.kernel.org</td>
        </tr>
                <tr>
            <td rowspan="2"> /platform/porting/connectivity.md</td>
            <td><em>https://wiki.iotivity.org/iotivity_security</em></td>
        </tr>
        <tr>
            <td> => I couldn't find the updated URL.</td>
        </tr>
                <tr>
            <td rowspan="2"> /application/tizen-studio/common-tools/emulator-features.md</td>
            <td><em>https://qemu.weilnetz.de/doc/qemu-doc.html#sec_005finvocation</em></td>
        </tr>
        <tr>
            <td> => https://qemu.weilnetz.de/doc/latest/</td>
        </tr>
                <tr>
            <td rowspan="2"> /application/tizen-studio/common-tools/emulator-features.md</td>
            <td><em>https://qemu.weilnetz.de/doc/qemu-doc.html#sec_005finvocation</em></td>
        </tr>
        <tr>
            <td> => https://qemu.weilnetz.de/doc/latest/</td>
        </tr>
    </tbody>       
</table>

### Bugs Fixed ###

 - Issue #1135

### CODE REVIEW

File: docs/application/tizen-studio/common-tools/emulator-features.md [hunk 1]
```diff
@@ -422,7 +422,7 @@ You can override the defined qemu values on the emulator starting time. For exam
    ./emulator --conf <configuration file> --ram 1024
    ```
 
-For more information on the start-up options, see [QEMU Emulator User Documentation](https://qemu.weilnetz.de/doc/qemu-doc.html#sec_005finvocation) and the `vm_launch.conf` file.
+For more information on the start-up options, see [QEMU Emulator User Documentation](https://qemu.weilnetz.de/doc/latest/) and the `vm_launch.conf` file.
```
Comment (journey2w) [inline docs/application/tizen-studio/common-tools/emulator-features.md:425]: The link for Start-up options is https://qemu.weilnetz.de/doc/latest/system/invocation.html.

https://qemu.weilnetz.de/doc/latest/ is unclear. If you want to use this URL, please change "For more ..." phrase.

File: docs/application/toc_all.md [hunk 2]
```diff
@@ -129,8 +129,8 @@
 ### Media and Camera
 #### [Overview](/application/dotnet/guides/multimedia/overview.md)
 #### [Media Content and Metadata](/application/dotnet/guides/multimedia/media-content-metadata.md)
```
Comment (journey2w) [inline docs/application/toc_all.md:131]: When a node has children, the node must not have a link.

At this moment, when I click **Media Content and Metadata**  on https://docs1.stg.tizen.org/staging/1200/application/native/guides/multimedia/overview/,  the page loses its path on LNB.

If you want to keep the page, you have to the toc as follow:
```
#### Media Content and Metadata
##### [Overview](/application/dotnet/guides/multimedia/media-content-metadata.md)
##### [Media Content](/application/dotnet/guides/multimedia/media-content.md)
...
```

By the way, I think deleting this line is not bad. To do this, you have to change the overview.md file as well.
These files will be shown as follow:

`toc.all.md`
```
#### [Overview](/application/dotnet/guides/multimedia/overview.md)
#### [Media Content](/application/dotnet/guides/multimedia/media-content.md)
#### [Metadata](/application/dotnet/guides/multimedia/metadata.md)
```

`overview.md`
````
# Multimedia

Multimedia features introduce how you can handle multimedia resources, such as media files, images, audio, video, camera, and radio.

You can use the following multimedia features in your native applications:

- [Media Content](media-content.md)

  You can update database details due to file (or folder) creation or deletion. If a received file (or folder) does not exist in the file system, it is removed from the database. You can also retrieve a list of media folders, retrieve a list of media items, and monitor changes in the media database. You can search for specific media folders and read their information, and retrieve media folder content.

- [Metadata](metadata.md)

    You can manage audio file attributes, extract metadata from media files, and retrieve various information related to the MIME type.
 
- [Image Editing](image-edit.md)
  ...

```

File: docs/application/tizen-studio/common-tools/emulator-features.md (hunk 1 above)
Comment (mijin85cho) [reply docs/application/tizen-studio/common-tools/emulator-features.md]: Fixed to link to https://qemu.weilnetz.de/doc/latest/system/invocation.html.

File: docs/application/toc_all.md (hunk 2 above)
Comment (mijin85cho) [reply docs/application/toc_all.md]: Fixed not to use media-content-metadata.md.

**Pull request merged**